### PR TITLE
Cart tab - RecyclerView to see Cart Items

### PR DESCRIPTION
**Pull request by jjojan**

### CODE REVIEW

File: app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    private static final String PRODUCT_DISPLAY = "Cart ";
+
+    private RecyclerView rvCart;
+    private CartAdapter fragmentAdapter;
+    private List<Cart> allCarts;
+
+    public CartFragment() {
+    }
+
+    public static CartFragment newInstance() {
```
Comment (kalalsherif) [inline app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java:39]: Why is this necessary when the constructor is public?

File: app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
```
Comment (kalalsherif) [inline app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java:50]: multiline: If you put any param on a second line, you should do it to all of them.

File: app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        allCarts = new ArrayList<>();
+        fragmentAdapter = new CartAdapter(getContext(), allCarts);
+        rvCart.setAdapter(fragmentAdapter);
+
+        queryCart();
+    }
+
+    private void queryCart() {
+        ParseQuery<Cart> query = ParseQuery.getQuery(Cart.class);
+        query.setLimit(20);
```
Comment (kalalsherif) [inline app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java:71]: Extract const: This makes it easier to understand where the value is coming from and change it in the future, especially if there get to be additional references.

File: app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java (hunk 1 above)
Comment (jjojan) [reply app/src/main/java/com/example/harvestfresh/fragments/CartFragment.java]: This was something done by default by android studio when generating a new fragment, fixed!
